StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/TextField/TextFieldElement+Card.swift

```diff
@@ -93,6 +108,9 @@ extension TextFieldElement {
                     return String.Localized.your_card_number_is_incomplete
                 case .invalidBrand, .invalidLuhn:
                     return String.Localized.your_card_number_is_invalid
+                case .disallowedBrand(let brand):
+                    let cardBrandDisplayName =  STPCardBrandUtilities.stringFrom(brand) ?? "This card brand"
```

Comment:
I don't think this would localize cleanly — could we have a second string with the full "The selected brand is not allowed" message?

Reply:
Ah you're probably right! I updated this to use a hard coded string for English, and a generic one for non-english that will be localized.

Reply:
Oh got it, fixed!